GLOSSARY.md

```diff
@@ -0,0 +1,86 @@
+## access point
+Networking hardware which creates a local area network over Wifi
+
+## analog
+In an electrical signal, existing in a continuous range, as opposed to the discrete ranges of digital
+
+## API
+Application program interface: a set of tools and protocols which can be used to build software. In the case of Tessel, the hardware API is a library you can use to connect to modules, manipulate LEDs, and call out specific pins
+
+## CLI
+Command line interface: a tool which lets you interact with a program (or in the case of Tessel, with the Tessel) via text-based commands in the command line, or terminal
+
+## digital
+In an electrical signal, existing at discrete values: on or off, zero or one
+
+## GPIO
+General purpose input and output: electrical pins which can be configured for various purposes
+
+## I2C
+A hardware communication protocol which uses two wires: SCL and SDA. Read more about it in the Communication Protocols doc
+
+## interrupt
+An electrical signal which tells the processor that something needs immediate attention
+
+## LAN
+Local area network
+
+## LED
+Light-emitting diode
+
+## master
+A machine or device directly controlling another machine or device
+
+## MDNS
+Multicast domain name system, a way to map IPs to devices in a small network
+
+## MISO
+Master input/slave output: in a SPI interface, the wire which carries commands to the controlling device from the device(s) under control
+
+## MOSI
+Master output/slave input: in a SPI interface, the wire which carries commands from the controlling device to the device(s) under control
+
+## module port
+A standard interface for connecting wires and modules to Tessel, configured as 10 "pins" or points of electrical connection
+
+## pins
+Points of electrical connection with an integrated circuit
+
+## ports
+Interfaces for electrical connections
+
+## PWM
+Pulse-width modulation: a hardware communication protocol where there is a digital signal fluctuating between values at a set period, and the message is encoded by changing the percentage of that period in which the signal stays at one of the values. This can be used to approximate an analog signal on a pin which only supports digital signaling
+
+## RX
+Receive: the line in a UART interface which receives data from the controlled device
```

Comment:
"wire" was used in earlier definitions, should we standardize on "wire" over "line"?